pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -276,14 +295,17 @@ func newServiceForCR(cr *pipelinesv1alpha1.GitopsService) *corev1.Service {
 		},
 	}
 	svc := &corev1.Service{
-		ObjectMeta: objectMeta(cr.Name, cr.Namespace),
-		Spec:       spec,
+		ObjectMeta: objectMeta(cr.Name, cr.Namespace, func(o *metav1.ObjectMeta) {
+			o.Annotations = map[string]string{
+				"service.beta.openshift.io/serving-cert-secret-name": cr.Name,
```

Comment:
Could you point me to docs for this, please?

Reply:
https://docs.openshift.com/container-platform/4.5/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate